extensions/cli/cli.go

```diff
@@ -0,0 +1,91 @@
+package cli
```

Comment:
This looks good. Markus, could you check helm and rkecli clients, do you think this should be another client? I think we could have this as a client. And these functions as extensions still, but they would depend on that client. What do you think?

Reply:
I'm unsure what part would be a client? Are you saying you think just everything in this shepherd PR should be in a separate client package instead?

Reply:
Resolved